web/beacon-app/src/features/topics/components/TopicQueryResult/index.tsx

```diff
@@ -0,0 +1,30 @@
+import ResultHeader from './ResultHeader';
+
+interface TopicQueryResultProps {
+  result: any;
+  isFetching?: boolean;
+}
+
+const TopicQueryResult = ({ result, isFetching = false }: TopicQueryResultProps) => {
+  const { data, error } = result;
+
+  if (isFetching) {
+    return <div>Loading...</div>;
+  }
+
+  if (error) {
+    return <div>Error: {error.message}</div>;
```

Comment:
```suggestion
    return <div>Error: {error?.message}</div>;
```

Reply:
I added this dummy implementation as a reminder to implement the error handling and loading process in the future.